test/lib/mayaUsd/render/mayaToHydra/testDagChanges.py

```diff
@@ -20,85 +20,91 @@
 import mayaUtils
 import mtohUtils
 
-class TestDagChanges(mtohUtils.MayaHydraBaseTestCase):
+from string import digits
+
+def fullPath(nodeName):
+    return cmds.ls(nodeName, l=True)[0]
+
+class DagChangesBaseTestCase(mtohUtils.MayaHydraBaseTestCase):
+
+    IMAGE_DIFF_FAIL_THRESHOLD = 0.01
+    IMAGE_DIFF_FAIL_PERCENT = 0.2
+
+    def rprimPath(self, mayaPath):
+        return '/MayaHydraViewportRenderer/rprims/Lighted' + mayaPath.replace('|', '/')
+
+class TestDagChanges(DagChangesBaseTestCase):
     _file = __file__
 
     def setUp(self):
         self.makeCubeScene()
 
         self.grp1 = cmds.createNode('transform', name='group1')
-        self.grp1Rprim = self.rprimPath(self.grp1)
-
         self.grp2 = cmds.createNode('transform', name='group2')
-        self.grp2Rprim = self.rprimPath(self.grp2)
 
-        self.imageVersion = None
-        if maya.mel.eval("defaultShaderName") != "standardSurface1":
-            self.imageVersion = 'lambertDefaultMaterial'
+        self.imageVersion = maya.mel.eval("defaultShaderName").rstrip(digits)
```

Comment:
Now using the same directory structure as the BasicRenderTest, with a subdirectory per default material.